js/build.js

```diff
@@ -26,9 +26,11 @@ Fliplet.Widget.instance('image-gallery', function(data) {
     if (!Fliplet.Env.get('interact')) {
       $(WALL_SELECTOR + ' .brick img').click(function() {
         var $clickedBrick = $(this)[0].parentElement;
-        Fliplet.Navigate.previewImages(data, {
+        data.options = {
```

Comment:
Where does `data` come from? What if `data` already has an `options` object?

This might be better if you don't know whether `data.options` already exists.

```js
data.options = data.options || {}
data.options.index = $clickedBrick.index - 1
```

Reply:
@tonytlwu Just committed the change. `data` comes from `Fliplet.Widget.instance('image-gallery', function(data)`